src/components/editor/AppController.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { browserHistory } from 'react-router';
+import FalcorController from "lib/falcor/FalcorController";
+import { setAppReady } from "lib/falcor/falcorUtils";
+
+export default class AppController extends FalcorController {
+  constructor(props) {
+    super(props);
+    this.safeSetState({
+      defaultValue: this.handleDefaultValue.bind(this)()
+    })
+    
+  }
+  static getFalcorPath() {
```

Comment:
I guess I can make it a non-falcor controller later in the process when I can completely rule in or out whether I need falcor calls. But yeah well spotted, I'll keep it in mind.

For now I haven't structured the access to articles around issues but just purely chronologically through when they were published in Ghost. I guess I could restructure it though? I think it works okay for now though